liukang/Blog/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+<head>
+ <link rel="stylesheet" type="text/css" href ="index.css">
+ <title>������ҳ</title>
+</head>
+<body>
+  <header class="page-header">
+   
+     <div class="header-name" >
+        <a href="index.html">My Blog</a>
+     </div>
+     <div class="header-list">
+        <a href="index.html" class="a-list">��ҳ</a>  
+        <a href="index.html" class="a-list">���˼��</a>       
+        <a href="index.html" class="a-list">����</a>
+     </div>
+  
+  </header>
+<section>
+  <nav class="article-index">
+   <ul>
+<li ><a href><b>HOME</b></a></li>

```

Comment:
```suggestion
<li ><a href='#'>HOME</a></li>
```
不要使用 b 标签创建粗体文本，使用 CSS 进行修改 [MDN](https://developer.mozilla.org/zh-CN/docs/Web/HTML/Element/b)